lib/pbench/agent/tool_meister_start.py

```diff
@@ -801,15 +800,43 @@ def start(_prog: str, cli_params: Namespace) -> int:
             ", ".join(_orchestrate_choices),
         )
         return ReturnCode.INVALIDORCHESTRATE
+    if cli_params.orchestrate == "create":
+        orchestrate = True
+    else:
+        assert (
+            cli_params.orchestrate == "existing"
+        ), f"Unexpected --orchestrate parameter, {cli_params.orchestrate!r}"
```

Comment:
Given the test at line 796, isn't this assertion overkill?  (Alternately, if you're really concerned about the value of not-`"create"`, perhaps the test at line 803 should be checking for not-`"existing"` instead?)

Reply:
Just moved existing code.  PRs to follow up with a change welcome!

Reply:
👃 🤏 

You "just" moved the existing code to a place where the assertion is no longer quite so valuable....